packages/design-system/src/organisms/tokenSelectV2/tokenSelectV2.test.tsx

```diff
@@ -0,0 +1,72 @@
+import { fireEvent, render, screen } from '@testing-library/react-native'
+import { TryrollTestProvider } from '../../providers'
+import { TokenSelectV2 } from '.'
+
+describe('TokenSelect', () => {
+  it('picks tokens', async () => {
+    const onChange = jest.fn()
+    const options = [
+      { name: 'Ether', symbol: 'ETH', value: 'eth', address: '0x00' },
+      { name: 'Wrapped Ether', symbol: 'WETH', value: 'weth', address: '0x01' },
+      { name: 'USD Coin', symbol: 'USDC', value: 'usdc', address: '0x02' },
+    ]
+    render(<TokenSelectV2 options={options} onChange={onChange} />, {
+      wrapper: TryrollTestProvider,
+    })
+
+    const selectInput = await screen.findByTestId('tokenSelectInput')
+    expect(selectInput).toBeDefined()
+
+    fireEvent.press(selectInput)
+
+    const searchInput = await screen.findByTestId('tokenSelectSearchInput')
+    expect(searchInput).toBeDefined()
+    options.forEach((option) => {
+      expect(
+        screen.getByTestId(`tokenSelectOption__${option.value}`),
+      ).toBeDefined()
+    })
+
+    fireEvent.changeText(searchInput, 'eth')
+    expect(screen.queryByTestId('tokenSelectOption__usdc')).toBeNull()
+
+    fireEvent.press(screen.getByTestId('tokenSelectOption__eth'))
+
+    expect(
+      await (
+        await screen.findByTestId('tokenSelectInput')
+      ).props.value,
+    ).toBe('Ether')
+    expect(onChange).toHaveBeenCalledWith('eth')
+  })
+
+  it('filters by address', async () => {
```

Comment:
Let's also check that:

1. Filtering by name & symbol works.
2. Placeholders & label are correct (check for existence in DOM).
3. Not found text is correct (when no options).
4. `closable` closes the select on press.